### PR TITLE
add activity links in Exec Sum

### PR DESCRIPTION
**Pull request by brendansudol**
addresses and closes: https://github.com/18F/cms-hitech-apd/issues/757

not much else to say :)

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: web/src/containers/ExecutiveSummary.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { connect } from 'react-redux';
+import PropTypes from 'prop-types'
```
Comment (mgwalker) [inline web/src/containers/ExecutiveSummary.js]: Different prettier config?  All the semicolons are gone.  😬

File: web/src/containers/ExecutiveSummary.js [hunk 2]
```diff
@@ -17,12 +18,22 @@ const ExecutiveSummary = ({ data, years }) => (
...
-              </div>
+            {d.id !== 'all' ? (
+              <Fragment>
+                <div className="h5">
+                  {t('activities.namePrefixAndNum', { number: i + 1 })}
+                </div>
+                <a
+                  href={`#activity-${d.id}`}
+                  className="h3 bold black"
+                  onClick={() => expandSection(d.id)}
```
Comment (mgwalker) [inline web/src/containers/ExecutiveSummary.js]: Nice

File: web/src/containers/ExecutiveSummary.js (hunk 1 above)
Comment (brendansudol) [reply web/src/containers/ExecutiveSummary.js]: oops, fixing

Comment (brendansudol) [reply web/src/containers/ExecutiveSummary.js]: fixed

**Pull request merged**